packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -317,9 +324,62 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
   }
 
   a11yStatusTimeout: any
+  charKeysPressedTimeout: any
 
   componentWillUnmount() {
     clearTimeout(this.a11yStatusTimeout)
+    clearTimeout(this.charKeysPressedTimeout)
+  }
+
+  /**
+   * Used to compute the filtered items (by value and search query) and, if needed,
+   * their string equivalents, in order to be used throughout the component.
+   * @param props
+   * @param state
+   */
+  static getDerivedStateFromProps(props: DropdownProps, state: DropdownState) {
+    const { items, itemToString, multiple, search } = props
+    const { value } = state
+
+    if (!items) {
+      return state
```

Comment:
```suggestion
      return null
```

> getDerivedStateFromProps is invoked right before calling the render method, both on the initial mount and on subsequent updates. It should return an object to update the state, or null to update nothing.

https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops